example/flake.nix

```diff
@@ -0,0 +1,67 @@
+{
+  description = "Example clojure project";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs";
+    cloure-nix-locker.url = "github:bevuta/clojure-nix-locker";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, clojure-nix-locker, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+        my-clojure-nix-locker = clojure-nix-locker.lib.customLocker {
+          inherit pkgs;
+          command = "${pkgs.clojure}/bin/clojure -T:build uber";
+          lockfile = "./deps.lock.json";
+          src = ./.;
+        };
+      in rec {
+        packages.uberjar = pkgs.stdenv.mkDerivation {
+          pname = "clojure-nix-locker-example";
+          version = "0.1";
+
+          src = ./.;
+
+          nativeBuildInputs = with pkgs; [
+            makeWrapper
+            clojure
+            git
+          ];
+
+          buildPhase = ''
+            source ${my-clojure-nix-locker.shellEnv}
+
+            # Now compile as in https://clojure.org/guides/tools_build#_compiled_uberjar_application_build
+            clojure -T:build uber
```

Comment:
Now I'm bothered by the WETness of this being the same as `command`. I'm not sure it's really WETness though, since they could in theory be different things (if you knew there was a reliable way to produce the dependencies that was faster)